### PR TITLE
Add steps for running code from an open PR

### PR DESCRIPTION
**Pull request by AlexLakatos**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,4 +12,14 @@ We're always open to pull requests, but these should be small and clearly descri
...
  
+## Reviewing a Pull Request
 
+To run the code for an open PR, follow these steps:
+
+1. `git clone https://github.com/Nexmo/nexmo-node.git`
+2. `cd nexmo-node`
+3. `git checkout BRANCH_NAME`
+4. `npm install`
+5. Update `examples/run-examples.js` to uncomment the examples you want to run
```
Comment (tbedford) [inline CONTRIBUTING.md]: 5. Uncomment the appropriate line in `examples/run-examples.js` for each example you want to run.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,4 +12,14 @@ We're always open to pull requests, but these should be small and clearly descri
...
+6. Update `example.env` in the examples folder with your api key and secret, and rename it to be `.env`
```
Comment (tbedford) [inline CONTRIBUTING.md]: 6. In the `examples` folder, copy `example.env` to `.env` and update it with your API key and API secret.

**Review by tbedford — CHANGES_REQUESTED** [review]
Minor suggestions.

**Comment by AlexLakatos** [comment]
@tbedford thanks for the suggestions, I've incorporated them in the PR.

**Review by tbedford — APPROVED** [review]

**Pull request merged**